objgraph.py

```diff
@@ -1208,43 +1221,51 @@ def _gradient(start_color, end_color, depth, max_depth):
     return h, s, v
 
 
-def _edge_label(source, target, shortnames=True):
+def _edge_labels(source, target, shortnames=True):
     if (_isinstance(target, dict)
             and target is getattr(source, '__dict__', None)):
-        return ' [label="__dict__",weight=10]'
+        return [' [label="__dict__",weight=10]']
     if _isinstance(source, types.FrameType):
         if target is source.f_locals:
-            return ' [label="f_locals",weight=10]'
+            return [' [label="f_locals",weight=10]']
         if target is source.f_globals:
-            return ' [label="f_globals",weight=10]'
+            return [' [label="f_globals",weight=10]']
```

Comment:
I tried using `yield` but it breaks the control flow around `return`

Reply:
oh I see yes that might be much better

Reply:
It would require replacing all `return` statements with `yield`, and then adding `return` with no arguments for short-circuiting the control flow (or rewriting the series of top-level if/if/if into `if/elif/elif').

Not worth it.  When I first suggested `yield`, I thought there would be more places with possible duplicate labels, but it looks like just this one place.

Reply:
lol I already did it